In-house patch.
Adds a newline on the end of the .in file to avoid the autotools failing on it.
Community will be asked if they will to add newline also in upstream.

--- a/foomatic-kitload.in	2016-07-25 16:32:15.439702509 -0700
+++ b/foomatic-kitload.in	2016-07-25 16:32:36.754761478 -0700
@@ -116,4 +116,6 @@
 sub show_errors {
     warn "\nError messages of the copying process:\n\n";
     print STDERR `cat $destdir$libdir/kitload2.log`;
-}
\ No newline at end of file
+}
+
+
